cspell.json

```diff
@@ -0,0 +1,61 @@
+{
+  "language": "en",
+  "ignorePaths": [
+    "bun.lockb",
+    "*.css",
```

Comment:
There shouldn't be any css files.
Also there shouldn't be bun.lockdb since project uses npm.
Can you also check that all of the listed locations are actually present. Otherwise if we keep adding unused code it becomes harder to maintain the project.